muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/actions/main-function/users/index.ts

```diff
@@ -102,8 +138,36 @@ let applyStaffmember: ApplyStaffmemberTriggerType = function applyStaffmember (d
   }
 }
 
+let updateStaffmember: UpdateStaffmemberTriggerType = function updateStaffmember(data) {
+  return async (dispatch: (arg: AnyActionType) => any, getState: () => StateType) => {
+    try {
+      await promisify(mApi().user.staffMembers.update(data.staffmember.identifier, data.staffmember), 'callback')().then(() => {
+
+        mApi().organizationUserManagement.staffMembers.cacheClear();
+
+        setTimeout(async () => {
```

Comment:
Because edit : muikku --> pyramus ---> muikku --> and if it just refreshes, it's too fast and no changes in state. I made it 2 second just to be sure.